src/Program.cs

```diff
@@ -27,112 +26,29 @@ internal class Program
         internal const int UnableToLocateMSBuildExitCode = 3;
         internal const int UnableToLocateDotNetCliExitCode = 4;
 
+        private static ParseResult? s_parseResult;
+
         private static async Task<int> Main(string[] args)
         {
-            var rootCommand = CreateCommandLineOptions();
+            var rootCommand = FormatCommand.CreateCommandLineOptions();
+            rootCommand.Handler = CommandHandler.Create(typeof(Program).GetMethod(nameof(Run)));
+
+            // Parse the incoming args so we can give warnings when deprecated options are used.
+            s_parseResult = rootCommand.Parse(args);
 
-            // Parse the incoming args and invoke the handler
             return await rootCommand.InvokeAsync(args);
         }
 
-        internal static RootCommand CreateCommandLineOptions()
+        public static async Task<int> Run(string? project, string? folder, string? workspace, string? verbosity, bool check, string[] include, string[] exclude, string? report, bool includeGenerated, IConsole console = null!)
```

Comment:
It may finally be time to wrap this line

Reply:
Curses!

I'll get it done.